terraform/application/application.tf

```diff
@@ -16,6 +16,7 @@ module "application_configuration" {
       ENVIRONMENT_NAME   = var.environment
       PGSSLMODE          = local.postgres_ssl_mode
       CANONICAL_HOSTNAME = local.canonical_hostname
+      APP_BASE_URL       = local.app_base_url
```

Comment:
Is this a good way to set the URL base to use for the magic link?
If it is, then a future optimisation would be to replace `DFE_SIGN_IN_REDIRECT_BASE_URL`, `TID_BASE_URL` and `ONELOGIN_REDIRECT_BASE_URL` with this `APP_BASE_URL`.